src/sspr_fmt_plug.c

```diff
@@ -23,6 +23,8 @@ john_register_one(&fmt_sspr);
 #include <omp.h>
 #endif
 
+#define OMP_SCALE               1  // MKPC and OMP_SCALE tuned on Core i7-6600U
+
```

Comment:
I just realized we still "need" `#ifndef OMP_SCALE` around this. Not sure if it's relevant anymore but it'd be safer. I approved a couple of commits without this. Actually I think it's safe to say it's obsolete so this is fine.

Reply:
Let's see if these changes break @jfoug's builds. I can redo the patch series if that happens.